### ISSUE TITLE
Deleted folder/files management

### ISSUE THREAD

**Issue by GoogleCodeExporter** [waterxi/svnplot#29]
```
When a directory is deleted in the repository, the files that directory
contained is not taken into account for LoC/files counting (not deducted).

The "files type" chart should represent the repository at a given moment
(last rev by default) and should not take into account deleted files.


```

Original issue reported on code.google.com by `cyber.s...@gmail.com` on 27 Feb 2010 at 3:53

**Comment by GoogleCodeExporter** [comment]
Original comment by `nitinbh...@gmail.com` on 12 Mar 2010 at 4:54
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `nitinbh...@gmail.com` on 12 Mar 2010 at 4:54
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `nitinbh...@gmail.com` on 12 Mar 2010 at 5:01

**Comment by GoogleCodeExporter** [comment]
```
fixed version 0.6 
```

Original comment by `nitinbh...@gmail.com` on 5 Apr 2010 at 3:28
- Changed state: **Fixed**
